legend-engine-pure-code-compiled-core/src/main/resources/core/pure/router/preeval/preeval.pure

```diff
@@ -714,6 +747,29 @@ function <<access.private>> meta::pure::router::preeval::prevalGenericSimpleFunc
   );
 }
 
+function meta::pure::router::preeval::isGetAll(v:ValueSpecification[0..1]):Boolean[1]
+{
+  $v->match([
+    s:SimpleFunctionExpression[1] | $s.func->in([
+      getAll_Class_1__T_MANY_,getAllVersions_Class_1__T_MANY_,getAll_Class_1__Date_1__T_MANY_,
+      getAll_Class_1__Date_1__Date_1__T_MANY_,getAllVersionsInRange_Class_1__Date_1__Date_1__T_MANY_,
+      getAllForEachDate_Class_1__Date_MANY__T_MANY_
+    ]) || $s.parametersValues->first()->isGetAll(),

```

Comment:
should we put this list on a common place?  At a glance seems incomplete compared with what the router have:

https://github.com/finos/legend-engine/blob/master/legend-engine-pure-code-compiled-core/src/main/resources/core/pure/router/routing/router_routing.pure#L317

Also, there are few places we been changing from ->in() to do || given it performs better on paths that are on the critical path.